### ISSUE TITLE
No Release Tag Pushed since 4.1.0

### ISSUE THREAD

**Issue by rohanKanojia** [openshift/api#587]
Hi guys,

I'm from @fabric8io team. We maintain [Kubernetes and Openshift Client](https://github.com/fabric8io/kubernetes-client) which is used by a lot of people in the java ecosystem. We generate Kubernetes/Openshift model from go structs in api and origin. Right now our Openshift go struct model is pointing at v4.1.0[0], but I have noticed that no other tag has been pushed since then. Even Openshift 4.3.0 got released recently. 

I'm just curious why Openshift upstream maintainers are not pushing tags. It's not like we're blocked as we can update model to latest master revision. But it would be nice if we upgrade with respect to
Openshift releases. 

[0] https://github.com/fabric8io/kubernetes-client#compatibility-matrix

**Comment by rohanKanojia** [comment]
@deads2k @smarterclayton @bparees @soltysh : polite ping, Would appreciate it if you could share your thoughts on this.

**Comment by soltysh** [comment]
It's still in-progress, there are several dependencies we need to handle before we'll be able to publish those.

**Comment by marun** [comment]
So long as this repo remains untagged, the workaround can be `go get` with the desired branch name (e.g. master, release-x.x) and go.mod will pin to the SHA of the branch head.

**Comment by rohanKanojia** [comment]
Yeah, we're managing with that till now. But we can't provide accurate compatibility matrices with respect to openshift versions

https://github.com/fabric8io/kubernetes-client#compatibility-matrix

**Comment by kaovilai** [comment]
How long does this repo plan to remain untagged?
/reopen
/lifecycle frozen
